### ISSUE TITLE
Chaos CW CH15 Theologians

### ISSUE THREAD

**Issue by atropik** [WarEmu/WarBugs#7313]
Quest step 3. Demon doesn't pop up.

**Comment by Torquemadra** [comment]
script, unsupported at this time.

**Comment by Ryanfawkez** [comment]
Check behind the rock; NPC you're supposed to kill is there.

**Comment by StaH84** [comment]
Except for the script issue (Is it possible now? I think I saw one in High Pass) I counted 2 other:

1. Wrong tooltip - doesn't match the discription:
![Bjoorn_013](https://user-images.githubusercontent.com/18528095/54086893-07ddd780-434e-11e9-997a-8d6530c6c03c.jpg)

2. No big red marker on map where to return after blowing the Horn of Bone:
![Bjoorn_016](https://user-images.githubusercontent.com/18528095/54086926-65722400-434e-11e9-8c75-45c251c5a95e.jpg)
